### ISSUE TITLE
how can TSM get CoVE-IO Manifest ACPI？

### ISSUE THREAD

**Issue by king729** [riscv-non-isa/riscv-ap-tee-io#124]
i am confused about how TSM can get  CoVE-IO Manifest

**Comment by sameo** [comment]
@king729 This is implementation specific, but for example the CoVE-IO manifest could be loaded together with the TSM itself, by the platform RoT, into protected memory. After starting up, the TSM would then retrieve the manifest at a known or passed location within that TSM carved memory.
 
Another option would be for the TSM to request it from the RoT, although that's a little more complex.

**Comment by king729** [comment]
@sameo hi thanks Should the CoVE-IO manifest be updated accordingly when a PCIe device is hot-plugged?

**Comment by sameo** [comment]
> @sameo hi thanks Should the CoVE-IO manifest be updated accordingly when a PCIe device is hot-plugged?

No, the manifest is a description of the platform (IOMMUs, PCI topology, etc). Dynamic insertion and removal of PCI devices into the manifest described PCI topology is not defined in the manifest itself.
